ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -77,19 +93,21 @@ public void testThread() throws InterruptedException {
 
     Map<CommandId, CommandStatus> statusStore = statementExecutor.getStatuses();
     Assert.assertNotNull(statusStore);
-//    Assert.assertEquals(4, statusStore.size());
     Assert.assertEquals(CommandStatus.Status.SUCCESS, statusStore.get(topicCommandId).getStatus());
     Assert.assertEquals(CommandStatus.Status.SUCCESS, statusStore.get(csCommandId).getStatus());
-//    Assert.assertEquals(CommandStatus.Status.SUCCESS, statusStore.get(csasCommandId).getStatus());
-//    Assert.assertEquals(CommandStatus.Status.ERROR, statusStore.get(ctasCommandId).getStatus());
+    if (statusStore.containsKey(csasCommandId)) {
```

Comment:
This seems weird. The test has no control over the commands that are processed. Tests need to be deterministic

Reply:
Using mocks refactored this so it will be deterministic.